packages/template-ui/src/views/Home.vue

```diff
@@ -129,7 +130,19 @@ export default {
       else {
         // FIXME add API to query random content nodes
         console.debug(`Fetch ${this.carouselSlideNumber} random content nodes`);
-        this.loadingCarouselNodes = false;
+
+        window.kolibri.getContentByFilter({
+          extra: {
+            channel_id: this.channel.id,
+            random: Math.random().toString(),
+            kind_in: 'video,audio,document,exercise,html5,zim',
```

Comment:
To avoid a explicit list, it is better to use a `only_content: true` rather than a list. If going wth a list, it should be an actual array of constant strings, not a string.